domain/src/main/java/org/oppia/domain/exploration/ExplorationRetriever.kt

```diff
@@ -56,7 +56,7 @@ class ExplorationRetriever @Inject constructor(private val context: Context) {
 
   // Loads the JSON string from an asset and converts it to a JSONObject
   @Throws(IOException::class)
-  private fun loadJsonFromAsset(assetName: String): JSONObject? {
+  fun loadJsonFromAsset(assetName: String): JSONObject? {
```

Comment:
Here & below: I suggest moving these to JsonAssetRetriever and StateRetriever utilities, respectively, rather than exposing an unrelated API to callers of ExplorationRetriever.

Reply:
Done